cerberus/validator.py

```diff
@@ -1065,11 +1085,12 @@ def _validate_minlength(self, min_length, field, value):
     def _validate_nullable(self, nullable, field, value):
         """ {'type': 'boolean'} """
         if value is None:
-            if nullable:
-                return True
-            else:
+            if not nullable:
                 self._error(field, errors.NOT_NULLABLE)
-                return True
+            self._drop_remaining_rules(
```

Comment:
yeah, i thought about something like `Validator.rules_that_dont_validate_none_values` that can be set in custom validators like `Validator.priority_validations`.